### DISCUSSION TITLE
Is the Winmerge ignorecolumns.dll working in 2.16.10???

### DISCUSSION THREAD

**Discussion by OmegaUkDev** [WinMerge/winmerge#717] [Q&A] [answered]
I simply can't get Plugin IgnoreColumns to work on 2.16.10 (or 2.16.11 beta)

I have hundreds of stock files that are generated that i need to diff but i need to ignore the quantity fields which are in column 6 (F in WinMerge).  I have set the prediffer correctly i think but WinMerge still reports differences.  I don't really want to resort to sed to prechop the files although I will if i have to.

Columns are identified correctly by WinMerge and the correct column is identified with the correct difference, so it is delimiting the files correctly.  But it still things there are differences in column 6 (F) even though its told to ignore it by the settings below:

Tried :  
 - automatic / manual prediffer
 - unticked all but IgnoreColumns.dll
 - File Filters on IgnoreColumns.dll set to "\.txt$;\.csv$" to include both txt and csv files (all are comma delimited)
 - Plugin Properties on Ignorecolumns.dll set to 6 (also tried 1-10 just in case)
 - Result is differences still identified?

DaveC

**Comment by sdottaka** [comment, marked as answer]
The IgnoreColumns.dll plugin ignores strings in a specified range of character positions.

If you want to ignore certain columns in the CSV file, use the IgnoreFieldsComma.dll plugin instead of the IgnoreColumns.dll plugin.

**Reply by OmegaUkDev** [reply]
Superb thank you.  Can't believe i missed that!!
